README.md

```diff
@@ -4,6 +4,17 @@ Graphical representation of the general workflow:
 
 ![](https://github.com/ebi-gene-expression-group/garnett-cli/blob/master/garnett_pipeline.png)
 
+## Installation 
+Garnett-cli is installed through conda: 
+```
+conda create --name <garnett-env>
```

Comment:
Probably mention that you need to have Bioconda installed properly with the correct channels etc. Something I realised the other day testing something else with Anja. See https://github.com/ebi-gene-expression-group/experiment_metadata.